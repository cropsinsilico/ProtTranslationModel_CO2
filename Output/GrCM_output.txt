Glyma_ID	Description	No_in_Xhu_model	Initial_protein_content	mRNA_ele	mRNA_Amb	new_prot_levels_Amb	new_prot_levels_Ele	Ele:Amb
Glyma.01G010200	Phosphoribulokinase	13	4.815447396	0.954347547	1.0	1.33333334187	1.27246340022	0.954347544054
Glyma.01G026700	Glycine transaminase	124	4.576365858	0.91513799	1.0	1.33333334126	1.22018399066	0.915137987556
Glyma.02G228700	6-phosphofructo-2-kinase	59	0.007142857	0.884803608	1.0	1.33333333008	1.1797381426	0.884803609111
Glyma.03G027500	Transketolase	7,10	7.23992103	1.0	0.836354883	1.11513985887	1.33333334036	1.19566467807
Glyma.04G008300	FRUCTOSE-BISPHOSPHATE ALDOLASE 1, CHLOROPLASTIC-RELATED	0	1.312651261	0.963203144	1.0	1.3333333333	1.2842708587	0.963203144051
Glyma.04G011900	Glucose-1-phosphate adenylyltransferase	23	0.102632112	1.0	0.508070369	0.677427157269	1.33333333187	1.96823129625
Glyma.06G094300	Glucose-6-phosphate isomerase	21	0.039783002	1.0	0.78405116	1.04540154419	1.33333333179	1.27542697751
Glyma.06G323700	Sucrose-phosphate synthase	56	0.041627585	1.0	0.676089159	0.90145220989	1.3333333318	1.47909486179
Glyma.08G044100	Phosphoglucomutase	54	0.165829146	1.0	0.832471469	1.10996195637	1.33333333194	1.20124237078
Glyma.08G165500	Phosphoglycerate kinase	2	2.5117348	0.872650392	1.0	1.33333333622	1.1635338576	0.872650391316
Glyma.08G175800	(S)-2-hydroxy-acid oxidase 	121/1	0.99961711	0.931469408	1.0	1.33333333251	1.24195921038	0.931469408358
Glyma.08G214300	inorganic diphosphatase	25	0.15796129	1.0	0.779392086	1.03918944584	1.33333333193	1.28305126392
Glyma.08G302600	Serine-glyoxylate transaminase	122	3.578866379	0.956855171	1.0	1.3333333388	1.27580689741	0.956855169131
Glyma.09G015500	Phosphoglycolate phosphatase	112	17.95202781	1.0	0.997852965	1.33047066051	1.33333335311	1.00215163903
Glyma.10G059500	Triose-phosphate isomerase	0	0.027777778	1.0	0.853358136	1.13781084533	1.33333333178	1.1718409411
Glyma.10G066700	Fructose-bisphosphate aldolase	51	0.115636587	1.0	0.807173556	1.07623140565	1.33333333188	1.2388909345
Glyma.10G086600	Sucrose-phosphate phosphatase	57	0.026641654	1.0	0.757245529	1.00966070292	1.33333333178	1.32057564281
Glyma.10G265400	Ribose-5-phosphate isomerase	11	0.077034884	0.861238678	1.0	1.3333333303	1.14831823606	0.861238679003
Glyma.10G281400	Glycerate dehydrogenase	123	0.553026581	1.0	0.864795613	1.15306081588	1.3333333324	1.15634259186
Glyma.13G062700	Starch synthase	23	0.102632112	1.0	0.792697291	1.05692971901	1.33333333187	1.26151560306
Glyma.13G222300	glycine hydroxymethyltransferase	131	37.42118172	1.0	0.686836641	0.915782276872	1.33333337629	1.45595018594
Glyma.15G012500	Glycerate kinase	113	1.343210077	1.0	0.68570568	0.914274241044	1.33333333335	1.45835163399
Glyma.15G038100	Triose-phosphate isomerase	50	0.002944444	1.0	0.955670181	1.2742269049	1.33333333175	1.04638610802
Glyma.15G250800	(S)-2-hydroxy-acid oxidase	0	0.99961711	1.0	0.774085874	1.0321144986	1.33333333294	1.29184633561
Glyma.17G228800	glycine dehydrogenase (aminomethyl-transferring)	131	37.42118172	1.0	0.620701159	0.724151441533	1.16666670982	1.61108111219
Glyma.17G261700	Catalase(CAT, EC1.11.1.6)	122/2	0.001346901	0.999448904	1.0	1.33333333008	1.33259853708	0.999448905248
Glyma.19G046600	Ribulose-bisphosphate carboxylase	1	107.0869611	0.949650566	1.0	1.33333359082	1.26620088063	0.949650477081
Glyma.19G088400	Ribulose-phosphate 3-epimerase	12	0.038732394	0.903328743	1.0	1.33333333019	1.20443832261	0.903328744089
Glyma.19G106800	Glyceraldehyde-3-phosphate dehydrogenase (NADP+)	3	14.54213822	1.0	0.716879996	0.955840027736	1.33333334906	1.39493357713
