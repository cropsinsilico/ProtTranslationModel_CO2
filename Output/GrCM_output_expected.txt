Glyma_ID	Description	Initial_protein_contentAmb	Initial_protein_contentEle	mRNA_ele	mRNA_Amb	Ele:Amb	EleMutant:Amb	AmbMutant:Amb
Glyma.01G026700	GGAT	4.576365858	4.139831453	4.139831453	4.576365858	0.95111	0.951110468	1
Glyma.01G115900	"CAB29.1, Chloro"	1.27908	0.865265167	0.865265167	1.27908	0.82248	0.822480777	1
Glyma.01G180800	"PSBP1, Chloro"	1.58139	1.403779341	1.403779341	1.58139	0.94217	0.942171427	1
Glyma.02G064700	"CAB6, Chloro"	1.27908	0.907057933	0.907057933	1.27908	0.8421	0.841986261	1.000005203
Glyma.02G080800	"CAB1, Chloro"	1.27908	1.001245517	1.001245517	1.27908	0.78244	0.782954465	1.001393578
Glyma.02G094900	"PSBP Like-1, Chloro"	1.346985	1.125194053	1.125194053	1.346985	0.89724	0.857709754	1.00165907
Glyma.02G249600	"RCA, Chloro"	1.2	0.898773885	0.898773885	1.2	0.86544	0.865435288	1
Glyma.02G282500	"PSBP1, Chloro"	1.264275	0.80288324	0.80288324	1.264275	0.7969	0.796902925	1
Glyma.02G309500	LHCA3	1.27908	1.164749113	1.164749113	1.27908	0.95426	0.954261363	1
Glyma.03G068100	"RCA, Chloro"	1.2	1.154641742	1.154641742	1.2	0.98092	0.980918677	1
Glyma.03G114600	"PSBP1, Chloro"	1.07397	0.905513456	0.905513456	1.07397	0.91823	0.918229792	1
Glyma.03G123100	PPD6	1.346985	0.891141228	0.891141228	1.346985	0.81338	0.813377002	1
Glyma.03G185800	T3PI	0.027777778	0.022236115	0.022236115	0.027777778	0.89471	0.89470673	1
Glyma.03G230300	PSBP	1.346985	1.174047282	1.174047282	1.346985	0.93358	0.933435916	1.00001356
Glyma.03G253500	"VDE, Chloro"	0.163	0.153720088	0.153720088	0.163	0.97112	0.971116895	1
Glyma.03G262300	LHCA2	1.27908	0.689311441	0.689311441	1.27908	0.73411	0.734106195	1
Glyma.04G014500	"PPD3, Chloro"	1.346985	0.884791339	0.884791339	1.346985	0.81047	0.810473934	1
Glyma.04G015900	GAPDH	14.54213822	11.43885112	11.43885112	14.54213822	0.8869	0.886904931	1
Glyma.04G020300	"PETE, Chloro"	1.4292036	1.407892071	1.407892071	1.4292036	0.99252	0.992516261	1
Glyma.04G051200	GDH	37.42118172	32.56776973	32.56776973	37.42118172	0.9329	0.932900359	1
Glyma.04G066000	ATPSynthase	4.645454545	4.20564011	4.20564011	4.645454545	0.95148	0.951484999	1
Glyma.04G112800	PSI_PSAK	0.405	0.340070132	0.340070132	0.405	0.91634	0.916340187	1
Glyma.04G167900	"CAB4, Chloro"	1.27908	0.686177035	0.686177035	1.27908	0.73244	0.732435246	1
Glyma.04G209900	"PPD2, Chloro"	1.346985	1.018580035	1.018580035	1.346985	0.86855	0.860385912	1.000338322
Glyma.04G215800	"PSAN, Chloro"	0.829305	0.702417889	0.702417889	0.829305	0.84596	0.84710036	1.001325738
Glyma.04G241000	"PPD4, Chloro"	1.346985	0.866608081	0.866608081	1.346985	0.8021	0.802102717	1
Glyma.04G242000	Thioredoxin	5.85	5.520602452	5.520602452	5.85	0.97144	0.971438483	1
Glyma.04G249700	"PSBS, Chloro"	1.26036	1.263317599	1.263317599	1.26036	1.00117	1.001172632	1.000000025
Glyma.05G022900	"PSAIII, Chloro"	1.087785	0.685685779	0.685685779	1.087785	0.79395	0.79394613	1
Glyma.05G025300	Ru5PE	0.038732394	0.024497293	0.024497293	0.038732394	0.79528	0.795283327	1
Glyma.05G048000	F-ATPase	1.327272727	1.095663639	1.095663639	1.327272727	0.90857	0.908570307	1
Glyma.05G112100	FDXC2	0.88695	0.606989045	0.606989045	0.88695	0.82726	0.827257775	1
Glyma.05G132600	"Thioredoxin O1, Mito"	5.85	4.882993514	4.882993514	5.85	0.91362	0.913619038	1
Glyma.05G149400	CAB	1.27908	0.757622718	0.757622718	1.27908	0.76962	0.769622295	1
Glyma.06G017900	GO	0.001346901	0.001253674	0.001253674	0.001346901	0.96477	0.964771514	1
Glyma.06G094300	G6PI	0.039783002	0.049825221	0.049825221	0.039783002	1.11912	1.119117897	1
Glyma.06G323700	SPS	0.041627585	0.072052887	0.072052887	0.041627585	1.31563	1.31563399	1
Glyma.07G019700	"PSAVI-1, Chloro"	0.69345	0.591096133	0.591096133	0.69345	0.92325	0.923254606	1
Glyma.07G028500	Diphosphatase	0.15796129	0.211354179	0.211354179	0.15796129	1.15672	1.156724898	1
Glyma.07G142700	FBPase	5.281887818	4.414585373	4.414585373	5.281887818	0.91422	0.849582907	0.928042594
Glyma.08G044100	"PGM, Cyto"	0.165829146	0.214437833	0.214437833	0.165829146	1.13716	1.137156591	1
Glyma.08G074000	CAB	1.27908	0.482336519	0.482336519	1.27908	0.61408	0.614081655	1.000000005
Glyma.08G087100	"Thioredoxin O1, Mito"	5.85	2.953705067	2.953705067	5.85	0.71057	0.710567976	1
Glyma.08G088000	TRX	1.863	1.653993852	1.653993852	1.863	0.94224	0.942237789	1
Glyma.08G097300	GO	0.99961711	1.328996635	1.328996635	0.99961711	1.15304	1.153041929	1
Glyma.08G126800	"FDX1, Chloro"	0.88695	0.842497172	0.842497172	0.88695	0.97462	0.974618514	1
Glyma.08G165400	PGK	2.5117348	2.72759739	2.72759739	2.5117348	1.04209	1.042085233	1
Glyma.08G165500	PGK	2.5117348	2.346879313	2.346879313	2.5117348	0.96663	0.902862855	0.933557571
Glyma.08G173700	"PSBR, Chloro"	0.65637	0.638717136	0.638717136	0.65637	0.98646	0.986461006	1
Glyma.08G302600	SGAT	3.578866379	3.365587497	3.365587497	3.578866379	0.96975	0.969745339	1
Glyma.09G087700	"PSAK, Chloro"	0.59427	0.461413038	0.461413038	0.59427	0.88116	0.881156456	1
Glyma.09G154700	"CAB26, Chloro"	1.27908	0.951294693	0.951294693	1.27908	0.8624	0.862399871	1
Glyma.09G250800	"PSAXI, Chloro"	1.032165	0.681551407	0.681551407	1.032165	0.8126	0.812596122	1
Glyma.09G255200	HPR	0.553026581	0.594500599	0.594500599	0.553026581	1.03682	1.036819473	1
Glyma.10G042000	"PSAIV-A, Chloro"	0.695565	0.638385201	0.638385201	0.695565	0.95802	0.95801552	1
Glyma.10G086600	SPP	0.026641654	0.036772905	0.036772905	0.026641654	1.17485	1.174852588	1
Glyma.10G161600	TRP26	5.85	5.381113888	5.381113888	5.85	0.95909	0.959087341	1
Glyma.10G210700	PGM	0.165829146	0.113547014	0.113547014	0.165829146	0.82748	0.82747985	1
Glyma.10G249000	"PSAII-1, Chloro"	0.241695	0.218963241	0.218963241	0.241695	0.95181	0.951813309	1
Glyma.10G268500	Aldolase	1.312651261	1.717945896	1.717945896	1.312651261	1.14401	1.292916549	1.157887695
Glyma.10G293500	Transketolase	7.23992103	6.659257767	6.659257767	7.23992103	0.95906	0.891670763	0.927916767
Glyma.11G100800	"HCF136, Chloro"	0.12852	0.104382643	0.104382643	0.12852	0.90122	0.901215777	1
Glyma.11G169700	PFK	0.007142857	0.006488433	0.006488433	0.007142857	0.95309	0.953090047	1
Glyma.11G226900	SBPase	0.264155289	0.254390311	0.254390311	0.264155289	0.98134	0.981342545	1
Glyma.12G199400	PET	0.7075224	0.660848732	0.660848732	0.7075224	0.96645	0.966453434	1
Glyma.12G202500	"PSB27-1, Chloro"	0.847575	0.442078381	0.442078381	0.847575	0.72093	0.710443238	1.000265209
Glyma.12G215100	"PSBQ Like - 2, Chloro"	1.118385	0.652923277	0.652923277	1.118385	0.76407	0.764073978	1
Glyma.13G028200	psbA	1.752165	1.416192719	1.416192719	1.752165	0.9028	0.903810132	1.037342494
Glyma.13G204800	APTC1	2.781818182	2.212152912	2.212152912	2.781818182	0.89175	0.891750171	1
Glyma.13G222300	GHMT	37.42118172	46.64974598	46.64974598	37.42118172	1.11652	1.116518421	1
Glyma.14G067000	"RCA, Chloro"	1.2	0.971066061	0.971066061	1.2	0.89957	0.899567517	1
Glyma.15G012500	GLK	1.343210077	0.952027569	0.952027569	1.343210077	0.84189	0.841885039	1
Glyma.15G114600	Cytochrome B	2.0124	1.625512577	1.625512577	2.0124	0.89875	0.898748157	1
Glyma.15G136200	Ri5PI	0.077034884	0.064798538	0.064798538	0.077034884	0.91715	0.917146868	1
Glyma.16G168000	FBPase	0.465302167	0.395812911	0.395812911	0.465302167	0.92231	0.857731084	0.929830579
Glyma.17G015600	AGPase	0.102632112	0.100289833	0.100289833	0.102632112	0.98852	0.988523097	1
Glyma.19G017200	"G6PI, Cyto"	1.727272727	0.970755903	0.970755903	1.727272727	0.74968	0.749677648	1
Glyma.19G022900	SS	0.102632112	0.070707018	0.070707018	0.102632112	0.83002	0.734908009	0.881138977
Glyma.19G046800	RuBisCO	107.0869611	95.42820347	95.42820347	107.0869611	0.944	0.943995832	1
Glyma.19G089100	PRK	4.815447396	6.327979294	6.327979294	4.815447396	1.14634	1.146341997	1
Glyma.19G251000	"VDE, Chloro"	0.163	0.095991709	0.095991709	0.163	0.7674	0.767402235	1
